src/notation/internal/notationinteraction.cpp

```diff
@@ -1971,6 +1971,30 @@ bool NotationInteraction::dropRange(const QByteArray& data, const PointF& pos, b
         return false;
     }
 
+    if (segment->isChordRestType()) {
+        bool allowDrop = true;
+        for (EngravingItem* item : segment->elist()) {
+            if (!item) {
+                continue;
+            }
+            ChordRest* chordRest = toChordRest(item);
+            if (chordRest->tuplet() && chordRest->tick() != chordRest->topTuplet()->tick()) {
+                //MScore::setError(MsError::DEST_TUPLET);
+                allowDrop = false;
+                break;
+            }
+        }
+
+        if (!allowDrop) {
+            apply();
```

Comment:
Why is it needed here?

Reply:
Cause if I don't call that function I get this problem that the drop region doesn't go away after finishing the drop.

https://github.com/user-attachments/assets/979db354-c159-4634-b8ab-47e81b80f18f

Reply:
Perhaps it's only necessary to call notifyAboutDropChanged()

Reply:
Ah yes you're right. Done!

Reply:
Last nitpick (sorry): we should call notifyAboutDropChanged() after endDrop(). Because endDrop() changes the state of the score and notifyAboutDropChanged() causes the score to be redrawn

Reply:
Good point. Maybe we should do the same thing a few lines below then? Cause that also currently does apply() followed by endDrop().

Reply:
Yeah, but perhaps we should keep apply() there and only change the order of the calls for now

Reply:
👍 Done!